frontend/src/scenes/pipeline/batch-exports/BatchExportEditForm.tsx

```diff
@@ -179,6 +179,26 @@ export function BatchExportsEditFields({
                             <LemonInput placeholder="e.g. posthog-events/" />
                         </LemonField>
 
+                        <div className="flex gap-4">
+                            <LemonField name="file_format" label="Format" className="flex-1">
+                                <LemonSelect
+                                    options={[
+                                        { value: 'JSONLines', label: 'JSON lines' },
+                                        { value: 'Parquet', label: 'Apache Parquet' },
+                                    ]}
+                                />
+                            </LemonField>
+
+                            <LemonField
+                                name="max_file_size_mb"
+                                label="Max file size (MiB)"
+                                showOptional
+                                className="flex-1"
+                            >
+                                <LemonInput type="number" min={0} />
+                            </LemonField>
```

Comment:
Could we add an "info" field saying something like: "Files over this max file size will be split into multiple files. Leave empty for no splitting regardless of file size". Or something like that? Just want to clarify what happens if it's not set.

Reply:
good suggestion; have added that now